apps/docs/integrations/aws-ses.mdx

```diff
@@ -45,10 +45,10 @@ export default function Email(props) {
 
 Import the email template you just built, convert into a HTML string, and use the AWS SES SDK to send it.
 
-```js index.tsx
+```js index.ts
```

Comment:
It needs to be `tsx` otherwise it will complain about the Email component.

Reply:
@bukinoshita You are right. I forgot about that. Because of the babel package, you can now use the component as a function. I updated the docs, you now have a React and a NodeJS variant